keras/backend/load_backend.py

```diff
@@ -0,0 +1,124 @@
+from __future__ import absolute_import
```

Comment:
Please branch this file from `__init__.py` to make it easier to review (I think `git mv` should work)

Reply:
So git detects that it's just a rename. It's github which doesn't detect it when displaying the git diff. Please see #12105 where I do a `git mv` in one commit and created an `__init__.py` in another commit. The diff displayed by github is the same as the one display in this PR.